content/dstu2/document-reference.md

```diff
@@ -110,7 +110,43 @@ present.
    x-xss-protection → 1; mode=block
 </pre>
 
+## Operation docref
+
+Argonaut operation for querying DocumentReferences for the supplied parameters:
```

Comment:
Left as is, as this operation is intended to return a "bundle" with multiple document references, though we currently return the CCD.